code/modules/events/ion_storm.dm

```diff
@@ -54,11 +55,49 @@
 
 			log_game("Ion storm changed laws of [key_name(M)] to [english_list(M.laws.get_law_list(TRUE, TRUE))]")
 			M.post_lawchange()
+			
+		//Generate Cyborg law change
+	for(var/mob/living/silicon/robot/M in GLOB.alive_mob_list)
```

Comment:
This will instantly unsync all borgs, synced borgs should only have a chance of being unsynced

Reply:
Ive play tested it while being synced to an AI and this isnt the case, lawsync prevents the individual borgs laws from changing